### ISSUE TITLE
Not possible to add tool without triggering createResponse on tool call

### ISSUE THREAD

**Issue by xathis** [transitive-bullshit/openai-realtime-api#6]
I'd like to add a tool manually, without the `addTool` helper, because that helper will automatically trigger a `createResponse()`.
In the original OpenAI version I can add a tool via updateSession and omit the createResponse call.

**Comment by transitive-bullshit** [comment]
Good call; it should def be possible to `addTool` without modifying the conversation.
